site/docs/plugins/commands.md

```diff
@@ -5,10 +5,682 @@ next: false
+customize how your bot handles commands, giving you more flexibility.
+
+### ignoreCase
+
+By default commands will match the user input in a case-sensitive manner. Having
+this flag set, for example, in a command named `/dandy` will match `/DANDY` the
+same as `/dandY` or any other case-only variation.
+
+### targetedCommands
+
+When users invoke a command, they can optionally tag your bot, like so:
+`/command@bot_username`. You can decide what to do with these commands by using
+the `targetedCommands` config option. With it you can choose between three
+different behaviors:
+
+- `ignored`: Ignores commands that mention your bot's user
+- `optional`: Handles both commands that do and that don't mention the bot's
+  user
+- `required`: Only handles commands that mention the bot's user
+
+### prefix
+
+Currently, only commands starting with `/` are recognized by Telegram and, thus,
+by the [command handling done by the grammY core library](../guide/commands). In
+some occasions, you might want to change that and use a custom prefix for your
+bot. That is made possible by the `prefix` option, which will tell the Commands
+plugin to look for that prefix when trying to identify a command.
+
+If you ever need to retrieve `botCommand` entities from an update and need them
+to be hydrated with the custom prefix you have registered, there is a method
+specifically tailored for that, called `ctx.getCommandEntities(yourCommands)`.
+It returns the same interface as `ctx.entities('bot_command')`
+
+:::tip
+Commands with custom prefixes cannot be shown in the Commands Menu.
+:::
+
+### matchOnlyAtStart
+
+When [handling commands](../guide/commands), the grammY core library will only
+recognize commands that start on the first character of a message. The Commands
+plugin, however, allows you to listen for commands in the middle of the message
+text, or in the end, it doesn't matter! All you have to do is set the
+`matchOnlyAtStart` option to `false`, and the rest will be done by the plugin.
+
+## RegExp Commands
+
+This feature is for those who are really looking to go wild, it allows you to
+create command handlers based on Regular Expressions instead of static strings,
+a basic example would look like:
+
+```js
+myCommands
+  .command(
+    /delete_([a-zA-Z]{1,})/,
+    (ctx) => ctx.reply(`Deleting ${ctx.message?.text?.split("_")[1]}`),
+  );
+```
+
+This command handler will trigger on `/delete_me` the same as in `/delete_you`,
+and it will reply `Deleting me` in the first case and `Deleting you` in the
+later, but will not trigger on `/delete_` nor `/delete_123xyz`, passing trough
+as if it wasn't there.
+
+You can use custom prefixes and localize them as usual.
 
 ## Plugin Summary
 
 - Name: `commands`
 - [Source](https://github.com/grammyjs/commands)
-- Reference
+- [Reference](/ref/commands/)
+- [Telegram Docs commands reference](https://core.telegram.org/bots/features#commands)
```

Comment:
Please remove this, the plugin summary should always consist of the same three values. It can be linked elsewhere on the page.